app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -32,6 +35,35 @@ class DradisDatatable {
             text: '<input type="checkbox" id="select-all" />',
             titleAttr: 'Select all'
           },
+          {
+            text: 'Delete',
+            className: 'btn-danger d-none',
+            name: 'bulkDeleteBtn',
+            action: function (event, dataTable, node, config) {
```

Comment:
Let's move this function to the class otherwise it makes the DataTables more convoluted than it is. Once defined, you'll be able to use:

```
action: this.bulkDelete.bind(this)
```

Reply:
Will do, thanks